pkgs/applications/blockchains/polkadot/default.nix

```
@@ -51,6 +52,15 @@ rustPlatform.buildRustPackage rec {
     };
   };
 
+  cargoPatches = [
+    # NOTE: bump `time` dependency to be able to build with rust 1.80
+    # should be removed on the next release
+    (fetchpatch {
+      url = "https://github.com/paritytech/polkadot-sdk/pull/5149.patch";
```

This should link to the commit's patch, since this PR is merged.

```suggestion
      url = "https://github.com/paritytech/polkadot-sdk/commit/326342fe63297668d74fc69095090e53d43a2b0a.patch";
```

---

If the PR is merged, this makes no difference. We only must fetch a commit if the PR is not merged.